### DISCUSSION TITLE
Finally finished remaking my website in Quarto!

### DISCUSSION THREAD

**Discussion by ForBo7** [quarto-dev/quarto-cli#3367] [Show and Tell]
And I’ve completed my new website! Uses quite a few Quarto features too, from tab sets to CSS grids to margin headers and much more!

The landing page features a dynamic dual listing: one listing shows my blog posts while the other listing shows my web apps. Making the window smaller stacks the lists instead. And for the web apps, I’ve embedded the ones I’ve created with Gradio and hosted on Huggingface, so I don’t need to learn JavaScript for the time being. The about me page features tab sets to reduce clutter too.

I plan to add more features in the future, mainly through Quarto extensions, such as a scroll-to-the-top button, text animations, and image lightboxes — the latter which I plan to use when I shift my 3D CG works to my website.

Quarto is cool! :smile:

Check it out here 👉 [https://forbo7.github.io](https://forbo7.github.io)

![](https://user-images.githubusercontent.com/85829792/202634843-b75c62b1-a76f-483f-a1d5-78363a5226d3.png)
_I've zoomed out the page so it's fully visible for this screenshot._

**Comment by neeldhara** [comment]
This looks awesome! Haven't looked at the articles yet, but really nice touches with icons in the navigation and other small details :) 

Looking forward to checking out the playground + the blog.

**Reply by ForBo7** [reply]
Thank you! 😃
